test/parallel/test-runner-run.mjs

```diff
@@ -178,6 +178,32 @@ describe('require(\'node:test\').run', { concurrency: true }, () => {
     });
   });
 
+  it('should include test type in enqueue, dequeue events', async () => {
+    const stream = await run({
+      files: [join(testFixtures, 'default-behavior/test/suite_and_test.cjs')],
+    });
+
+    stream.on('test:enqueue', common.mustCall((data) => {
+      if (data.name === 'this is a suite') {
+        assert.strictEqual(data.type, 'suite');
```

Comment:
There is no guarantee that any of these assertions will run, even with the `common.mustCall()` (for example, if the test fixture was changed in the future). I would suggest adding a test plan.

Reply:
Sounds good, but I'm not familiar with what a test plan is. Is there an example I could reference?

Reply:
Oh, you meant this: https://nodejs.org/api/test.html#contextplancount

Good call. I added a plan and it uncovered two of the assertions weren't working due to a leading space in the name.